datafusion/core/tests/fuzz_cases/window_fuzz.rs

```diff
@@ -50,6 +50,23 @@ use rand::rngs::StdRng;
 use rand::{Rng, SeedableRng};
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 16)]
+async fn tmp_test() -> Result<()> {
+    let session_config = SessionConfig::new();
+    let ctx = SessionContext::new_with_config(session_config);
+
+    let result = ctx.sql("SELECT x, count(x) OVER (ORDER BY x NULLS FIRST ROWS BETWEEN UNBOUNDED PRECEDING AND 1 PRECEDING) AS c FROM (VALUES (1), (2), (NULL), (3)) t(x)").await?.collect().await;
+    assert!(result.is_err());
+    // When this fails, drop the commit that introduce this test. This will unignore temporarily disabled tests.
+    result
+        .err()
+        .unwrap()
+        .message()
+        .contains("is declared as non-nullable but contains null values");
```

Comment:
Should there be another `assert!` around this `result....contains()` check?

Reply:
keen eye. thanks for catching me fall